### PR TITLE
Documenting workaround for brewed python; added symlink formula

### PR DESCRIPTION
**Pull request by GreenGremlin**
I reversed the pydistutils changes and added a symlink formula.

The symlink formula includes creating parent directories of the link, if they don't already exist. It also removes the link when the environment is removed. Optionally it will be removed when an environment is deactivated if the active_only option is True.

### CODE REVIEW

File: sprinter/formula/eggscript.py [hunk 1]
```diff
@@ -59,7 +59,7 @@ def __install_eggs(self, config):
         if config.has('egg'):
             eggs += [config.get('egg')]
         if config.has('eggs'):
-            eggs += [egg.strip() for egg in re.split(',|\n', config.get('eggs'))]
+            eggs += [egg.strip() for egg in re.split(',(?!<)|\n', config.get('eggs'))]
```
Comment (GreenGremlin) [inline sprinter/formula/eggscript.py]: Shouldn't split on version ranges i. e. `package>=1.1.1,<1.2a`

File: scripts/sandbox.sh [hunk 2]
```diff
@@ -5,25 +5,40 @@ error () {
     echo $1
     exit $last_error
 }
+INITIAL_DIR=`pwd`
 
+: ${SPRINTER_REPO:=toumorokoshi}
```
Comment (toumorokoshi) [inline scripts/sandbox.sh]: can we call this "sprinter_github_user" instead? I would've guessed "sprinter_repo" refers to the repo name itself, and not the account owner.

File: uranium [hunk 3]
```diff
@@ -4,13 +4,22 @@
...
 # * run uranium
 import sys
-URANIUM_STANDALONE_URL = "https://raw.githubusercontent.com/toumorokoshi/uranium/master/uranium/scripts/uranium_standalone"
+import os
 
-try:
-    from urllib2 import urlopen as urlopen
-except:
-    from urllib.request import urlopen as urlopen
+URANIUM_PATH = os.getenv('URANIUM_PATH', None)
```
Comment (toumorokoshi) [inline uranium]: nice! these might be nice in the main uranium repo too, if you're interested :)

**Comment by toumorokoshi** [comment]
the changes look good. if you can adjust the environment variable names, I'm happy to merge.

File: uranium (hunk 3 above)
Comment (GreenGremlin) [inline uranium]: I believe I added them in the PR I closed, I'll verify and reopen.

**Comment by GreenGremlin** [comment]
I update the variable names, went with GITHUB_ACCOUNT. I also updated Uranium, I'll reopen that PR as well.

**Comment by GreenGremlin** [comment]
I noticed the eggscript formula was failing silently. I updated it to make it less passive aggressive.

**Comment by toumorokoshi** [comment]
thanks!

**Pull request merged**